migration/migrations/course/20180703121911_forum_thread_status.py

```diff
@@ -1,10 +1,11 @@
 def up(config, conn, semester, course):
```

Comment:
This change is needed to run the regrade_per_assignment migration: without changing this file, migrations will fail before regrade_per_assignment is reached.

Reply:
I'd argue that this belongs in its own PR.  If you have a commit where you only changed this migration file, cherry pick that into a new branch, otherwise make a new commit that changes just this file.  I'll review and approve that quickly